docs/advanced/retry-operations.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Retrying Operations
+order: 4
+---
+
+# Retrying Operations
+
+The `retryExchange` lets us retry specific operation, by default it will
+retry only `network errors` but we can specify additional options to add
+functionality.
+
+## Installation and Setup
+
+First install `@urql/exchange-retry` alongside `urql`:
+
+```sh
+yarn add @urql/exchange-retry
+# or
+npm install --save @urql/exchange-retry
+```
+
+You'll then need to add the `retryExchange`, exposed by this package, to your `urql` Client:
+
+```js
+import { createClient, dedupExchange, cacheExchange, fetchExchange } from 'urql';
+import { retryExchange } from '@urql/exchange-retry';
+
+// None of these options have to be added, these are the default values.
+const options = {
+  initialDelayMs: 1000,
+  maxDelayMs: 15000,
+  randomDelay: true,
+  maxNumberAttempts: 2,
+  retryIf: err => err && err.networkError,
+};
+
+const client = createClient({
+  url: 'http://localhost:1234/graphql',
+  exchanges: [
+    dedupExchange,
+    cacheExchange,
+    fetchExchange,
+    retryExchange(options), // Use the retryExchange factory to add a new exchange
```

Comment:
This is added in the wrong position and will need to be moved ahead of `fetchExchange` 😇 it may also be worth adding a note that the exchange should be added after any `cacheExchange` and in front of `fetchExchange` while referencing the appropriate section on the “Exchanges” page about ordering